### ISSUE TITLE
[SF][ACC] Label Colon Not announce on NVDA

### ISSUE THREAD

**Issue by IrvinDelgado** [SAP/ui5-webcomponents#6488]
### **Bug Description**
The label colon is not announce in NVDA when `showColon = true`

### **Expected Behavior**
If label is "Name" with `showColon=true` then it should be announced as "Name : (announcing the word colon)"

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Label/ 
2. Use NVDA to read label with `showColon=true`
3. Watch as only the label is announce

### **Isolated Example**
If possible, please provide an isolated example by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

### **Context**
- UI5 Web Components version: 1.1.0
- OS/Platform: Windows
- Browser: Chrome
- Affected component: Label

### **Log Output / Stack Trace / Screenshots**

![Screenshot 2023-02-13 at 3 52 24 PM](https://user-images.githubusercontent.com/62973847/218601538-0a739498-088e-45eb-8270-09fe0e6051f4.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

When the "showColon" property is set to true and colon is visualised, it is not announced by JAWS/NVDA while navigating with VPC cursor mode. 
I checked the OpenUI5 sap.m.Label in the same scenario and the colon is announced.

Could you please look over the issue?

Thank you in advance!

Kind Regards,
Niya

**Comment by TeodorTaushanov** [comment]
Hi @IrvinDelgado,

The proper use of ui5-label is to define a label for elements like input, textarea, select. In that case the ui5-label is read when the focus goes to the bound element. For that the ":" (colon) should not be read, as it is just a decorative text. It should not be read even with VPC cursor mode.

ui5-label is not supposed to be used alone. In that case a regular ui5-text can be used.

Best,
Teo

**Comment by olannyv** [comment]
Internal ticket -2995 created to improve ui5-label docu & samples
